superset/assets/src/dashboard/stylesheets/builder-sidepane.less

```diff
@@ -13,10 +13,17 @@
   }
 
   .trigger {
-    height: 18px;
-    width: 25px;
+    font-size: 16px;
+    margin-top: 3px;
```

Comment:
would using flexbox to vertically center this be a more robust approach instead of precise margins that would break if parent height changes?

Reply:
use flexbox.